doc/user/content/overview/open-cdc.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Materialize Open CDC"
+description: "The Open CDC format allows Materialize to detect when records are duplicated or out of order."
```

Comment:
This is a true statement, but it probably isn't the most direct description? Like, I might expect "The Open CDC format allows Materialize to communicate changes to collections of data, even when records are ...".

Reply:
The page is renamed materialize-cdc.md, so the changes aren't showing up here. I did revise the phrasing for this in both the page description and the opening paragraph.